### DISCUSSION TITLE
rowview.php - picture device no charge

### DISCUSSION THREAD

**Discussion by alex001x** [opendcim/openDCIM#1458] [Q&A]
hello,
When I have 1 container, the device images are displayed, but when I have 2 or more containers the device images no longer load in opendcim 23.03.
I cleaned the SQL database
Removed special characters
Clear cache explorer firefox
cache rebuild in opendcim
Separate folder Assets + chmod 755 folders and chown -R www-data:www-data 
ubuntu localization assets: /opt/openDCIM/assets
ubuntu localization opendcim: /opt/openDCIM/opendcim
Help me !

**Comment by wilpig** [comment]
Unable to reproduce.

http://dev.opendcim.org/rowview.php?row=3

This is a row that is nested several levels down in subcontainers.  Did you get all the updates past 23.03 or just stock?  There is a large api issue related to pictures not loading #1435

**Reply by alex001x** [reply]
RESOLVED - thks you
